app/models/auth_source.rb

```diff
@@ -40,6 +40,14 @@ def auth_method_name
     "Abstract"
   end
 
+  def ldap?
+    AuthSource.is_a? AuthSourceLdap
```

Comment:
I believe, we can use polymorphism for this as well. Set this to `false` and overwrite the method in `app/models/auth_sources/auth_source_ldap.rb`